service/src/main/java/bio/terra/workspace/common/utils/AwsUtils.java

```diff
@@ -808,7 +808,6 @@ public static void stopSageMakerNotebook(
     }
   }
 
-  // TODO(BENCH-727): make idempotent
```

Comment:
Is this change related to this PR?

Reply:
Not exactly, this was a stale comment. Bench-727 is already fixed